python/core/auto_generated/layout/qgslayoutitemscalebar.sip.in

```diff
@@ -228,6 +228,56 @@ transferred to the scalebar.
 
 .. seealso:: :py:func:`lineSymbol`
 
+.. versionadded:: 3.14
+%End
+
+    QgsFillSymbol *fillSymbol1() const;
+%Docstring
+Returns the primary fill symbol used to render the scalebar (only used for some scalebar types).
+
+Ownership is not transferred.
+
+.. seealso:: :py:func:`setFillSymbol1`
+
+.. seealso:: :py:func:`fillSymbol2`
+
+.. versionadded:: 3.14
+%End
+
+    void setFillSymbol1( QgsFillSymbol *symbol /Transfer/ );
```

Comment:
While we are cleaning up scale bars, what about changing those to setFillSymbol and setFillSymbolAlternate (or setFillAlternateSymbol, or setAlternateFillSymbol) instead of setFillSymbol{1,2}? 

That said, I recon it's a PITA to do the change for what is purely code aesthetic, feel free to ignore :)

Reply:
I'll pass, this pr has been a royal PITA to get through the ci as is 😜

Reply:
(might do in a follow up)

Reply:
Honestly, I can't blame you if you don't :) we can postpone to 4.0 too.